.vscode/tasks.json

```
@@ -0,0 +1,15 @@
+{
+  "version": "2.0.0",
+  "tasks": [
+    {
+      "label": "build:debug-env",
+      "type": "shell",
+      "command": "bash",
+      "args": [
+        "-lc",
+        "cat apps/service/.env.shared apps/service/.env.local > apps/service/.env.debug"
```

just realized since people might try to edit the .debug file if they don't know the difference between the 3 env files, we should probably add a warning at the top of the .debug file

and create .env.local if it doesn't exist to avoid errors

```suggestion
        "echo \"# DO NOT EDIT - Auto generated by VS Code task\" > apps/service/.env.debug && touch apps/service/.env.local && cat apps/service/.env.shared apps/service/.env.local >> apps/service/.env.debug"
```

---

added text in there.